projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -4399,7 +4399,7 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         }
 
 
-        const footerHeight = this.summariesHeight || this.tfoot.nativeElement.offsetHeight - this.tfoot.nativeElement.clientHeight;
+        const footerHeight = this.summariesHeight || this.footer.nativeElement.offsetHeight || this.tfoot.nativeElement.offsetHeight - this.tfoot.nativeElement.clientHeight;
```

Comment:
This will apply the footer height twice when paging is enabled and summaries are not - once from footerHeight which will be this.footer.nativeElement.offsetHeight and then again the same height  from this.getPagingHeight(), which also returns  this.footer.nativeElement.offsetHeight.

Reply:
Also there are some failing tests as a result of this change - https://travis-ci.org/IgniteUI/igniteui-angular/builds/589962111#L3806